### PR TITLE
Add link to Debug trait

### PR DESCRIPTION
**Pull request by swaroopch**
I don't recall reading about this `Debug` trait so far in the book. Please ignore this PR if I have missed that part.

r? @steveklabnik

### CODE REVIEW

**Comment by steveklabnik** [comment]
Thanks @swaroopch ! I want to make a chapter specifically about display and debug, but until then, this is an improvement for sure. Thanks!

File: src/doc/trpl/error-handling.md [hunk 1]
```diff
@@ -181,6 +181,8 @@ match version {
 This function makes use of an enum, `ParseError`, to enumerate the various
 errors that can occur.
 
+The [`Debug`](../core/fmt/trait.Debug.html) trait is what let's us print the enum value using the `{:?}` format operation.
```
Comment (sfackler) [inline src/doc/trpl/error-handling.md]: This should be "lets", not "let's", right?

Comment (steveklabnik) [inline src/doc/trpl/error-handling.md]: ah yes, and it probably should be to std, not core.

@swaroopch , up for a second PR after this one lands? heh. It's already being tested in the rollup, or i'd say fix it first

Comment (swaroopch) [inline src/doc/trpl/error-handling.md]: I added a second commit to same branch in my fork - https://github.com/rust-lang/rust/commit/f6e5369e53f5444240aa983c6e6be6725932e399 - but I'm a bit confused on how this works - should I do a separate PR?

Comment (steveklabnik) [inline src/doc/trpl/error-handling.md]: Ah! so, https://github.com/rust-lang/rust/pull/25377 failed. so no, you don't need to make a new PR
